src/sidebar/components/Annotation/Annotation.tsx

```diff
@@ -45,19 +32,34 @@ function SavingMessage() {
       )}
       data-testid="saving-message"
     >
-      <Spinner
-        classes={classnames(
-          'text-xl',
-          // Slowly fade in the Spinner such that it only shows up if
-          // the saving is slow
-          'animate-fade-in-slow'
-        )}
-        size="small"
-      />
+      <span
+        // Slowly fade in the Spinner such that it only shows up if the saving
+        // is slow
+        className="text-xl animate-fade-in-slow"
```

Comment:
The current fade-in is constant, right? We might want to tweak this in future so that there is a delay before any fade-in and then a more rapid fade-in afterwards.